tox.ini

```diff
@@ -0,0 +1,23 @@
+[tox]
+envlist = {py3}-sklearn{0.23},flake8
+
+[flake8]
+max-line-length = 120
+
+[testenv]
+deps =
+    sklearn0.23: scikit-learn==0.23.1
+    -r{toxinidir}/requirements.txt
+    -r{toxinidir}/test-requirements.txt
+conda_deps=
+    pyarrow=0.14.1
+    mlio-py=0.1
```

Comment:
Should we upgrade the version of ML-IO for this release? Any reasons for or against?

Reply:
I've used the same versions used in 0.23 upgrade so not sure if there were any reasons not to upgrade